src/box/iproto.cc

```diff
@@ -2138,6 +2138,47 @@ tx_process_rollback(struct cmsg *m)
 	tx_end_msg(msg, &header);
 }
 
+/*
+ * In case the request does not contain a space or identifier but contains a
+ * corresponding name, tries to resolve the name.
+ * A space identifier is mandatory for a DML request, so in case the space
+ * identifier is not present in the request and we fail to get an identifier
+ * from the space name, we must return with an
+ * error, setting a missing request field diagnostic, just as it was done in
+ * `xrow_decode_dml` before space names in IPROTO requests were introduced.
```

Comment:
Nit: This comment seems outdated - the code below doesn't set "missing request field diagnostic".

Reply:
Dropped this comment.